src/lib/services/user-service.ts

```
@@ -134,23 +135,37 @@ export class UserService {
      * Verify user credentials
      */
     static async verifyCredentials(email: string, password: string) {
-        const user = await this.getUserByEmail(email);
-
-        if (!user) {
-            return null;
-        }
-
         try {
-            const isPasswordValid = await argon2.verify(user.password, password);
+            const user = await this.getUserByEmail(email);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)